prometheus/promdb/promdb.go

```diff
@@ -0,0 +1,132 @@
+package promdb
```

Comment:
we should have package level docs in a doc.go file

Reply:
i'll have to look into that

Reply:
added